### ISSUE TITLE
Touch screen support behaves incorrectly on Windows 8.1 logon screen

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3903]
**Reported by k_kolev1985 on 2014-02-20 11:45**
I've noticed today on my ASUS T100 tablet that NVDA's touch screen support behaves strangely. On the logon screen (where I input my PIN), sometimes it does not work at all, sometimes it reports the things under my finger, but activating them is not possible (at least I couldn't activate the input field). Sometimes however, it behaves correctly - it's random. NVDA does not give any error sounds. The tablet runs Windows 8.1 32-bit, all updates installed. I'm running there the latest NVDA snapshot: next-10380,9ee9b11. With NVDA 2013.3 I did not experience such problems.

**Comment by leonardder** [comment]
@k-kolev1985: Is this still relevant to you?

**Comment by k-kolev1985** [comment]
@leonardder Yes, it seams still to occur in Windows 10 v1607. Creators Update hasn't yet arrived on this machine automatically and I haven't updated to it manually. But I've noticed that if I restart the machine (which forces it to do a cold boot instead of a fast startup), it behaves as expected in regards to touch interaction between NVDA and Windows.

**Comment by Adriani90** [comment]
@k-kolev1985 I guess this issue is still reproducible. Right?
@josephsl your thoughts are very appreciated. Thanks.

**Comment by k-kolev1985** [comment]
@Adriani90 I rarely use my touch screen computer these days, since I'm mainly on my desktop PC or on my phone. 🙂

However, I've tried to login into my touch screen computer (which is now running Windows 10 version 1809) now and here is what I've found.
1. I was able to activate most of the controls on the logon screen, except for the edit field for entering my PIN.
2. I was able to invoke the touch screen keyboard for entering my PIN when I double tapped not in the edit field for the PIN, but by double tapping outside it in an empty spot of the logon screen.
So, in conclusion, the issue with the edit field for the PIN may be a problem in Windows itself and not in NVDA. If anyone else can test this to confirm my observations it would be good.

P.S.: But after a restart, even double tapping outside the PIN edit field did not bring up the touch screen keyboard.

**Comment by k-kolev1985** [comment]
@Adriani90 If I don't forget, I can test it with Windows 10 latest update in the next days. Can't test it with Windows 11 - the machine is not compatible with it.

**Comment by k-kolev1985** [comment]
@Adriani90 As far as I could test it, now it works acceptably well. I suggest that we close this issue.

**Comment by Adriani90** [comment]
Thanks very much for testing. Closing as works for me.
